src/App.vue

```diff
@@ -504,21 +504,19 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
       return
     }
 
-    // now that we have user info and org info, store the completing party
+    // now that we have user info and org info, populate the completing party
+    // NB: these are all empty for staff
     this.setCompletingParty({
-      firstName: this.getUserFirstName,
-      middleName: '',
-      lastName: this.getUserLastName,
+      firstName: this.isRoleStaff ? '' : this.getUserFirstName,
+      lastName: this.isRoleStaff ? '' : this.getUserLastName,
       mailingAddress: {
         addressCity: this.getOrgInfo?.mailingAddress.city,
         addressCountry: this.getOrgInfo?.mailingAddress.country,
         addressRegion: this.getOrgInfo?.mailingAddress.region,
         postalCode: this.getOrgInfo?.mailingAddress.postalCode,
         streetAddress: this.getOrgInfo?.mailingAddress.street,
         streetAddressAdditional: this.getOrgInfo?.mailingAddress.streetAdditional
-      },
-      email: this.getUserEmail,
-      phone: this.getUserPhone
```

Comment:
Middle name, email and phone are legitimate Completing Party fields, IMHO, but they are not used at the moment. As they are optional fields (both in [CompletingPartyIF](https://github.com/bcgov/bcrs-shared-components/blob/main/src/interfaces/completing-party-interface.ts) and in the [party schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json)), if they are empty then they are deleted when this filing is created/filed.